app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -677,6 +677,37 @@ class Result extends React.Component {
     };
   };
 
+  nextRandomUnmarkedSubmission = () => {
```

Comment:
*This is a global comment.* I thought about the name for this action (naming is hard!) and came to the following conclusions:

1. Remove the word "next" since order should not be implied.
2. Change the word "Unmarked" to "Incomplete" to be consistent with the MarkUs state naming.